--- a/makefile
+++ b/makefile
@@ -16,12 +16,7 @@
 check-and-reinit-submodules :
 ifeq (,$(filter $(NOGOAL),$(MAKECMDGOALS)))
 ifeq (,$(findstring clean,$(MAKECMDGOALS)))
-	@if git submodule status 2>/dev/null | egrep -q '^[-]|^[+]' ; then \
-		echo "$(red)INFO: Need to reinitialize git submodules$(reset)"; \
-		git submodule update --init; \
-		echo "$(blue)Done$(reset)"; \
-	else echo "$(blue) Submodule up to date$(reset)"; \
-	fi
+	@echo "$(blue) Submodule check skipped$(reset)"
 endif
 endif
